www/js/app.min.js

```diff
@@ -1,377 +0,0 @@
-'use strict';
```

Comment:
If you're deleting this entire file and **not** adding it to the gitignore, that's silly, because next time someone commits it'll just be pushed right back on!